### PR TITLE
Mention missing dots in report volcano

### PR DESCRIPTION
**Pull request by WackerO**
Instead of https://github.com/nf-core/differentialabundance/pull/229

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by pinin4fjords — COMMENTED** [review]
Assume the GSEA changes here are because we need to merge in master.

Actually. what I was thinking with the warning is to actually check the data for values of 0 in the p values, and conditionally display a prominent warning under the plot:

"x features with p values of 0 are not shown- please refer to the results tables"

**Comment by WackerO** [comment]
> Assume the GSEA changes here are because we need to merge in master.
> 
> Actually. what I was thinking with the warning is to actually check the data for values of 0 in the p values, and conditionally display a prominent warning under the plot:
> 
> "x features with p values of 0 are not shown- please refer to the results tables"

Ah, those are in this PR because I created volcano_round from dev after adding the GSEA update.

Can also do that, but then I'll have to check for all values that are too low to fit in the plot, not just 0.

**Comment by WackerO** [comment]
How about this? @pinin4fjords 
[SRP254919 2.html.zip](https://github.com/nf-core/differentialabundance/files/14236252/SRP254919.2.html.zip)

File: nextflow.config [hunk 1]
```diff
@@ -31,7 +31,7 @@ params {
     report_contributors     = null
     report_description      = null
     report_scree            = true
-    report_round_digits     = 4
+    report_round_digits     = -1
```
Comment (pinin4fjords) [inline nextflow.config:34]: You want the rounding off by default now?

**Review by pinin4fjords — APPROVED** [review]
Looks good - thanks Oskar. Tiny question about default behaviour, but I'll leave you to resolve that (or not) before you merge.

File: nextflow.config (hunk 1 above)
Comment (WackerO) [reply nextflow.config:34]: Yeah that's an excellent question...I discussed this again with a colleague. I'm going to reenable the rounding by default for now. Let's see if this leads to further confusions with very small p values, then I might turn it off by default.

**Pull request merged**